lms/templates/problem.html

```diff
@@ -25,42 +25,44 @@ <h3 class="hd hd-2 problem-header" id="${ short_id }-problem-title" aria-describ
        />
       </div>
     % endif
-    <div class="problem-action-buttons-wrapper">
-    % if demand_hint_possible:
-    <span class="problem-action-button-wrapper">
-        <button type="button" class="hint-button problem-action-btn btn-default btn-small" data-value="${_('Hint')}" ${'' if should_enable_next_hint else 'disabled'}><span class="icon fa fa-question" aria-hidden="true"></span>${_('Hint')}</button>
-    </span>
-    % endif
-    % if save_button:
-    <span class="problem-action-button-wrapper">
-        <button type="button" class="save problem-action-btn btn-default btn-small" data-value="${_('Save')}">
-            <span class="icon fa fa-floppy-o" aria-hidden="true"></span>
-            <span aria-hidden="true">${_('Save')}</span>
-            <span class="sr">${_("Save your answer")}</span>
-        </button>
-    </span>
-    % endif
-    % if reset_button:
-    <span class="problem-action-button-wrapper">
-        <button type="button" class="reset problem-action-btn btn-default btn-small" data-value="${_('Reset')}"><span class="icon fa fa-refresh" aria-hidden="true"></span><span aria-hidden="true">${_('Reset')}</span><span class="sr">${_("Reset your answer")}</span></button>
-    </span>
-    % endif
-    % if answer_available:
-    <span class="problem-action-button-wrapper">
-        <button type="button" class="show problem-action-btn btn-default btn-small" aria-describedby="${ short_id }-problem-title"><span class="icon fa fa-info-circle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span></button>
-    </span>
-    % endif
-    </div>
+
     <div class="submit-attempt-container">
       <button type="button" class="submit btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" data-should-enable-submit-button="${ should_enable_submit_button }" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
           <span class="submit-label" aria-hidden="true">${ submit_button }</span><span class="sr">${_("Submit your answer")}</span>
```

Comment:
@staubina this is a quick fix that came up during manual testing.  JAWS has recently started to ignore ```aria-hidden``` on focusable/interactive elements (something I don't necessarily agree with).  So JAWS reads this button as "Submit Submit your answer".  Since Submit should be enough for all users sighted or not AND there is an argument to be made for speech input users that the visible label should match the accessible name, I think we should get rid of the SR text here (Submit your answer) and remove the ```aria-hidden``` attribute from the "Submit" span.  That way, all users get the same label.  I don't think its necessary to add the additional context here, especially now that we have the associated description text.